hydrolib/core/io/ini/util.py

```diff
@@ -82,15 +85,16 @@ def validate_correct_length(cls, values: dict):
             # length attribute not present, possibly defer validation to a subclass.
             return values
 
-        length = length + length_incr
+        length = max(length + length_incr, min_length)
```

Comment:
if `min_length` here is the greatest value, the code says that the lists must exactly have this number of values, but cannot have more than that. Is that correct? The key word says it is the minimal required length suggesting that the number of values is allowerd to be higher.

Reply:
Also SC is complaining about the cognitive complexity of this method. Mayme refactor a little by extracting the lower body of this function.

Reply:
I corrected the documentation/explanation + fixed code smell.